Sources/TrustlySDK/Utils/Constants.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Constants.swift
+//  Pods-TrustlySDK_Example
+//
+//  Created by Marcos Rivereto on 29/01/24.
+//
+
+import Foundation
+
+
+struct Constants {
+    static let buildSDK = "3.3.0"
+    static let baseDomain = "paywithmybank.com"
+    
+    static let RETURN_URL = "msg://return"
+    static let CANCEL_URL = "msg://cancel"
+    static let ESTABLISH_VERSION = "2"
+    static let DEVICE_TYPE = "mobile"
+    static let DEVICE_PLATFORM = "ios:native"
+    static let PAYMENTTYPE_VERIFICATION = "Verification"
+    
+    static let LIGHTBOX_CONTEXT_INAPP = "in-app-browser"
+    static let LIGHTBOX_CONTEXT_WEBVIEW = "web view"
```

Comment:
I think the correct string is 'webview'
`{"settings":{"integrationStrategy":"webview"}}`